docs/Testing.md

```diff
@@ -27,14 +27,30 @@ npm run test
   - install driver for chrome ``` appium driver install chromium ```
   - install driver for safari ``` appium driver install safari ```
 - Android Emulator for [Samsung Galaxy S23 Ultra](https://developer.samsung.com/galaxy-emulator-skin/guide.html) is configured or iOS Simulator for [iPhone 15 Pro / 15 Pro Max](https://developer.apple.com/documentation/xcode/installing-additional-simulator-runtimes)
-- Create .env file in your local root project folder with APP_PATH property with path to app build for chosen platform
+- Create .env file in your local root project folder with APP_PATH, KERIA_IP property with path to app build for chosen platform
 ```
 # Android
-# APP_PATH=<LOCAL_PATH/app-release-unsigned.apk>
+APP_PATH=<LOCAL_PATH/app-release-unsigned.apk>
+KERIA_IP=<IP_V4>
 
 # iOS
 APP_PATH=<LOCAL_PATH/App.app>
-```
+KERIA_IP=<IP_V4>
+```
+#### How to get IP v4 address:
```

Comment:
Add

> This is required to connect the simulator to the locally running KERIA docker container on your machine.

And add an instruction to make sure docker compose is running somewhere

Reply:
I would put this in the text, not the code. Just below "How to get IP v4 address"